Modules/Core/Transform/include/itkScaleTransform.hxx

```diff
@@ -261,8 +261,8 @@ ScaleTransform<TParametersValueType, VDimension>::ComputeMatrix()
 
 // Back transform a point
 template <typename TParametersValueType, unsigned int VDimension>
-inline typename ScaleTransform<TParametersValueType, VDimension>::InputPointType
-ScaleTransform<TParametersValueType, VDimension>::BackTransform(const OutputPointType & point) const
+auto
```

Comment:
Do we also want to remove all those `inline` keywords? I don't think they really matter, but the original author(s) seemed to care about them... 🤔

Reply:
You are absolutely right. It was search and replace oversight. I will amend the commit